### DISCUSSION TITLE
MQTT client minimum characters validation

### DISCUSSION THREAD

**Discussion by Asafb26** [arendst/Tasmota#22280] [Q&A]
Hi,
I have a MQTT server with more than 50 clients.
Trying to connect my tasmota device but I get an error about min characters limit for my MQTT password.
My MQTT password is 4 length characters, while its indeed not secured, why do the clients should validate the security level of my server? 
Is there any wat to bypass it without recompile the project?
Thanks

**Comment by sfromis** [comment]
The idea is that such a "password" is so ridiculously short that it does not count as "security". To me it makes more sense to allow the broker to accept connections without credentials.

Anyway, the length validation is only in the web UI, meaning that you can circumvent it using the `MqttPassword` command.

**Reply by Asafb26** [reply]
Great didn't know about the MqttPassword command. By the way, is it really necessary to have this limitation in the web UI if it's allowed in the console? It sounds a bit off, what do you think?

I agree that 4 characters is too short and doesn't qualify as "security" (the same goes for "password", "admin" or even "123456" IMO) but why bother checking it on the client side? A mistake has already been made (on the server) 

Imagine having a wifi password "paasword", but the client device returns an error "password could not be 'password'", which is indeed ridiculous, but not the client's responsibility.
